OrbitCore/LinuxTracingSession.h

```diff
@@ -7,14 +7,13 @@
 #include "ScopeTimer.h"
 #include "StringManager.h"
 #include "TcpServer.h"
-
 #include "absl/synchronization/mutex.h"
 
 // This class stores information about tracing session
 // and provides thread-safe access and record functions.
 class LinuxTracingSession {
  public:
-  explicit LinuxTracingSession(TcpServer* tcp_server);
+  explicit LinuxTracingSession(std::shared_ptr<TcpServer> tcp_server);
```

Comment:
Is TracingSession somehow lives longer than GTcpServer?

(This is fine for now if it is really necessary, mid-term this Session should stop depending on TcpServer)

Reply:
Yes, it turns out i did not know the best practices regarding when to use which pointer. 
* I changed this back to a "normal" pointer
* GTcpServer is now a unique_ptr, owned by OrbitApp

Thanks for the help!!